packages/core/src/blocks/TableBlockContent/TableExtension.ts

```diff
@@ -1,13 +1,40 @@
 import { callOrReturn, Extension, getExtensionField } from "@tiptap/core";
 import { columnResizing, tableEditing } from "prosemirror-tables";
+import { TableView } from "prosemirror-tables";
+import { Node as PMNode } from "prosemirror-model";
+import { mergeCSSClasses } from "../../util/browser.js";
 
 export const TableExtension = Extension.create({
   name: "BlockNoteTableExtension",
 
   addProseMirrorPlugins: () => {
+    class CustomTableView extends TableView {
```

Comment:
This is unrelated to the new buttons. It's just because the `columnResizing` extension adds a node view to the table block, which gets rendered out instead of our `render` implementation. This means there's no wrapping `blockContent` HTML element which is inconsistent with all the other blocks.

Reply:
i.e. just a bug fix? was it causing specific issues?